### PR TITLE
Give more guidance about recurring activities that aren't relationships

### PR DESCRIPTION
**Pull request by catherio**
From a coworker: "it would be super helpful to have an option for 'ongoing activity' that isn't a partner or housemate. like a multi-day meeting, or a movie shoot (which my sister is on right now), or thanksgiving for that matter. or maybe one could put in how many hours of exposure."

Give at least a littttttttle bit of guidance that extended stays are like housemates, and repeated few-hour interactions can just be added up.

### CODE REVIEW

**Comment by catherio** [comment]
Some of the changes are also to help users like this: https://twitter.com/Geo_Rex_H/status/1328135514935812097

**Comment by catherio** [comment]
workin' on that test now

File: src/locales/en.json
Comment (catherio) [reply src/locales/en.json:148]: Hm. My intuition is adding breadth won't help, and simplicity helps here. I want to keep this at a level where it's pretty clear what to imagine: someone acting like they live here.

In particular I want it to be fairly clear to users that this is a level of comfort in the space that assumes they are not wearing a mask, so users won't wonder where the mask option is. (They already sometimes do). Like if they're staying over, they clearly won't be masked when sleeping, eating, etc. Versus just "often around".

File: src/locales/en.json
Comment (catherio) [reply src/locales/en.json:129]: Agree

File: src/locales/en.json
Comment (catherio) [reply src/locales/en.json:133]: Do you have a better idea of how to help users like this, who thought the calculator was for "risk to me in general", not "risk to me from something specific I do"? https://twitter.com/Geo_Rex_H/status/1328135514935812097

Comment (catherio) [reply src/locales/en.json:133]: (apologies this was a more tacit part of the PR description)

Comment (catherio) [reply src/locales/en.json:133]: Work is not an "ongoing relationship", but it is a specific thing I do even though it is not a "one-time" activity

Comment (catherio) [reply src/locales/en.json:133]: "Is the thing we actually want to ask "do you spend more than 5 hours a week indoors with these people?" -> No, because it's not 5 hrs if masked, or ventilated, or ....  it's too confusing to try to explain when to "change over" to using the cap. I don't want to get into that in this PR.

Comment (beshaya) [reply src/locales/en.json:133]: I don't think this is going to improve that user's experience; changing "single" to "specific" doesn't help them figure out their total risk. If we want to help them, we need to explain that you need to enter all your activities and relationships for the last 9 days and add them up - I don't think we ever say that.

File: src/locales/en.json
Comment (beshaya) [reply src/locales/en.json:148]: Ok, you've convinced me on this one.

File: src/locales/en.json
Comment (beshaya) [reply src/locales/en.json:133]: Maybe run it by disc-usability-and-confusion for another opinion?

Comment (catherio) [reply src/locales/en.json:133]: I just meant "I assumed it was going to tell me what risk I (and my wife) were at in general. I was confused when it asked for specific incidents". So he wouldn't assume and wouldn't be confused. 

I did not mean "how do I add up risks to see how risky my behaviour is?". Obviously this does not help with that.

Comment (blanchardjeremy) [reply src/locales/en.json:133]: Hmmm. I don't have a strong opinion here. I don't think changing this one word (or keeping it) will really make that big of a difference. I think better to just get it out. Not sure how helpful that is. But there ya have it.

**Review by beshaya — APPROVED** [review]

File: src/locales/en.json
Comment (beshaya) [reply src/locales/en.json:133]: So let me get this straight:

User: "I was confused when it asked for specific incidents"
Us: We we will fix this by changing the wording to ask for "specific incidents"

It seems to me that the problem is that the user wants the calculator to be omniscient and magically know his risk. If we want to address that confusion, we'd need to know why he thought that. Are we advertising the calculator to do this? Is someone else saying it does? Or is this just a case of someone projecting their wants onto our product?

Anyway, I'd prefer it were "single" instead of "specific" but won't hold up the change over that word.

Comment (catherio) [reply src/locales/en.json:133]: In my mind it's like

User: "When you said burger, I assumed beef burger. I was confused when you gave me a lamb burger"
Us: We will fix this by changing the wording to say it is a "lamb burger".

I think the user was projecting their wants onto the product.

**Comment by beshaya** [comment]
In my mind it's like

User: "When you said 'go to a restaurant' I assumed you meant a vegan
restaurant. I was confused when all you served were burgers.
Us: "We will fix this by changing our menu to say 'beef burger'"

It doesn't really change the problem that he only came in because he
thought we were serving something we weren't.

**Pull request merged**
